sources/platform/integrations/langchain.md

```diff
@@ -57,22 +59,22 @@ loader = apify.call_actor(
 )
 ```
 
-_NOTE: The Actor call function can take some time as it loads the data from LangChain documentation website._
```

Comment:
can we make it into proper admonition with title?

Reply:
ok, look better with admonition.